payjoin/src/receive/optional_parameters.rs

```diff
@@ -103,15 +104,15 @@ impl Params {
             (Some(_), None) | (None, Some(_)) => {
                 warn!("only one additional-fee parameter specified: {params:?}");
             }
-            _ => (),
+            (None, None) => (),
```

Comment:
This is a notable change to prevent the mutant, seeing as it is the only remaining possibility I thought it was an ok change to make here.